assembl/static2/js/app/reducers/postsReducer.js

```diff
@@ -11,4 +11,56 @@ export const postsById = (state = Map(), action) => {
   return state;
 };
 
-export default combineReducers({ postsById: postsById });
\ No newline at end of file
+export const topPostSubject = (state = '', action) => {
+  switch (action.type) {
+  case 'UPDATE_TOP_POST_SUBJECT':
+    return action.topPostSubject;
+  default:
+    return state;
+  }
+};
+
+export const topPostBody = (state = '', action) => {
+  switch (action.type) {
+  case 'UPDATE_TOP_POST_BODY':
+    return action.topPostBody;
+  default:
+    return state;
+  }
+};
+
+export const topPostFormStatus = (state = false, action) => {
+  switch (action.type) {
+  case 'UPDATE_TOP_POST_FORM_STATUS':
+    return action.isTopPostFormActive;
+  default:
+    return state;
+  }
+};
+
+export const subjectRemainingChars = (state = 10000, action) => {
+  switch (action.type) {
+  case 'UPDATE_SUBJECT_REMAINING_CHARS':
+    return action.subjectRemainingChars;
+  default:
+    return state;
+  }
+};
+
+export const bodyRemainingChars = (state = 10000, action) => {
```

Comment:
These action methods all look the same, isn't there a way to factorize some of this code?